besu/build.gradle

```diff
@@ -94,4 +94,5 @@ dependencies {
   testImplementation 'org.awaitility:awaitility'
   testImplementation 'org.mockito:mockito-core'
   testImplementation 'commons-io:commons-io'
+  testImplementation 'org.testcontainers:testcontainers'
```

Comment:
where is this used?

Reply:
it was introduced via the TesseraTestHarness.